docs/Project-Board-Guide.md

```diff
@@ -17,13 +17,13 @@ Please familiarise yourself with our:
-| Ready for test | Contains PRs that are ready for the test engineers to test. | Once approved by a test engineer, it is their responsibility to update the PR with an approval and then the card will move to "Ready for merge". | n/a |
-| Ready for merge | Contains tickets that are approved by a test engineer. | If your PR has no CI (continuous integration) errors and the merge button is not disabled, then you can merge your PR and then it will automatically be moved to "Done". For cleanliness, you must also delete your branch after clicking merge. | n/a |
+| PR in progress | Contains cards that have been automatically added here once a PR is assigned to the project "Simorgh". | It is your responsibility to move the card to "Code Review", when you wish for it to be reviewed by others. Teams prioritise code reviews over actual development. | PRs should be created on first commit, so that the team can see what you are working on. Draft PRs are preferable until the code is ready for review, when it can be converted to a proper PR. |
+| Code review | Contains PRs that are deemed ready to be reviewed by other developers. | Once 2 other developers have approved your PR, it will automatically be moved to "Ready for merge". The initial developer or a code reviewer should decide if it needs manual testing, and if so will need to manually move it to the "Ready for test" column for a tester to pick up. If your PR requires "UX, A11Y, External Review", then you will have to manually move it to that column. | If you had paired with someone on an issue, then one participant can be the reviewer but the 2nd reviewer must be a code maintainer. |
+| UX, A11Y, External Review | Contains issues that require an accessibility swarm, UX or External review. | When the reviewing party approves the PR, the card will then have to be manually moved to "Ready for test" or "Ready for merge" depending on whether it needs manual testing. | [Read more about accessibility swarms](https://bbc-news.github.io/accessibility-news-and-you/accessibility-swarms) |
+| Ready for test | Contains PRs that are ready for the test engineers to test. | Once approved by a test engineer, it is their responsibility to update the PR with an approval and move the card to "Ready for merge". | n/a |
+| Ready for merge | Contains tickets that have been code reviewed | It is the respnsibility of the author or code reviewer to check at this stage if the ticket needs manual testing by a tester. If not, you can proceed with the merge process. If your PR has no CI (continuous integration) errors and the merge button is not disabled, then you can merge your PR and it will automatically be moved to "Done". For cleanliness, you must also delete your branch after clicking merge. | n/a |
```

Comment:
```suggestion
| Ready for merge | Contains tickets that have been code reviewed | It is the responsibility of the author or code reviewer to check at this stage if the ticket needs manual testing by a tester. If not, the author or code reviewer should proceed with the merge process. If your PR has no CI (continuous integration) errors and the merge button is not disabled, then you can merge your PR and it will automatically be moved to "Done". For cleanliness, you must also delete your branch after clicking merge. | n/a |
```